cards/card_component.js

```diff
@@ -16,6 +16,12 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
    * only if such a toggle is present in the DOM.
    */
   onMount() {
+    //polyfill for image styling (object-fit) in ie11
+    if (['product-prominentimage', 'product-prominentimage-clickable'].includes('{{componentName}}')) {
```

Comment:
is this going to be an issue if somebody forks the card with a custom name? could we just run the polyfill on the entire card all the time?

Reply:
you are right. updated!